llvm/test/CodeGen/RISCV/rvv/fixed-vectors-vror.ll

```diff
@@ -1924,18 +1964,34 @@ define <8 x i64> @vror_vv_v8i64(<8 x i64> %a, <8 x i64> %b) {
 }
 
 define <8 x i64> @vror_vx_v8i64(<8 x i64> %a, i64 %b) {
-; CHECK-LABEL: vror_vx_v8i64:
-; CHECK:       # %bb.0:
-; CHECK-NEXT:    vsetivli zero, 8, e64, m4, ta, ma
-; CHECK-NEXT:    vmv.v.x v12, a0
-; CHECK-NEXT:    li a0, 63
-; CHECK-NEXT:    vand.vx v16, v12, a0
-; CHECK-NEXT:    vsrl.vv v16, v8, v16
-; CHECK-NEXT:    vrsub.vi v12, v12, 0
```

Comment:
Looks like this causes us to miss the vrsub.vi pattern. But I think it + the subsequent vand.vx could all be done in the scalar domain?

Reply:
I think so. I think it all happens very late and we end up going to target nodes immediately so now generic transforms fire.